src/dashboard/src/components/archival_storage/views.py

```diff
@@ -253,57 +361,61 @@ def search_augment_file_results(es_client, raw_results):
     return modifiedResults
 
 
-def create_aic(request, *args, **kwargs):
-    aic_form = forms.CreateAICForm(request.POST or None)
-    if aic_form.is_valid():
-        aip_uuids = ast.literal_eval(aic_form.cleaned_data["results"])
-        logger.info("AIC AIP UUIDs: {}".format(aip_uuids))
+def create_aic(request):
+    """Create AIC from POST-ed list of AIP UUIDs.
 
-        # The form was passed a raw list of all AIP UUIDs mapping the user's query;
-        # use those to fetch their names, which is used to produce files below.
-        query = {"query": {"terms": {"uuid": aip_uuids}}}
-        es_client = elasticSearchFunctions.get_client()
-        results = es_client.search(
-            body=query,
-            index="aips",
-            _source="uuid,name",
-            size=elasticSearchFunctions.MAX_QUERY_SIZE,  # return all records
-        )
+    :param request: Django request object.
+    :return: Direct to ingest tab.
+    """
+    uuids = request.POST.get("uuids")
+    if not uuids:
+        messages.error(request, "Unable to create AIC: No AIPs selected")
+        return redirect("archival_storage_index")
 
-        # Create files in staging directory with AIP information
-        shared_dir = settings.SHARED_DIRECTORY
-        staging_dir = os.path.join(shared_dir, "tmp")
+    # Make list from comma-separated string of UUIDs
+    aip_uuids = uuids.split(",")
+    logger.info("AIC AIP UUIDs: {}".format(aip_uuids))
 
-        # Create SIP (AIC) directory in staging directory
-        temp_uuid = str(uuid.uuid4())
-        destination = os.path.join(staging_dir, temp_uuid)
-        try:
-            os.mkdir(destination)
-            os.chmod(destination, 0o770)
-        except os.error:
-            messages.error(request, "Error creating AIC")
-            logger.exception(
-                "Error creating AIC: Error creating directory {}".format(destination)
-            )
-            return redirect("archival_storage_index")
-
-        # Create SIP in DB
-        mcp_destination = destination.replace(shared_dir, "%sharedPath%") + "/"
-        databaseFunctions.createSIP(mcp_destination, UUID=temp_uuid, sip_type="AIC")
+    # Use AIP UUIDs to fetch their names, which is used to produce files below
+    query = {"query": {"terms": {"uuid": aip_uuids}}}
+    es_client = elasticSearchFunctions.get_client()
+    results = es_client.search(
+        body=query,
+        index="aips",
+        _source="uuid,name",
+        size=elasticSearchFunctions.MAX_QUERY_SIZE,  # return all records
```

Comment:
```suggestion
        size=elasticSearchFunctions.MAX_QUERY_SIZE, 
```
Really, only `MAX_QUERY_SIZE` so `10000`.

Reply:
Removed the comment since the line is self-describing